prestodb/hdp2.6-hive-kerberized/files/etc/krb5.conf

```diff
@@ -13,6 +13,10 @@
 
 [realms]
  LABS.TERADATA.COM = {
-  kdc = hadoop-master
+  kdc = hadoop-master:88
+  admin_server = hadoop-master
+ }
+ OTHERLABS.TERADATA.COM = {
```

Comment:
does the OTHERLABS's admin server allow to admin the OTHERLABS realm (like passwd)? `kadmind` configuration in supervisord doesn't say "OTHER"